app/views/admin/courses/_form.html.erb

```diff
@@ -14,5 +14,10 @@
         <%= f.text_field :slug %>
     </div>
 
+    <div class="field">
+        <%= f.label "Sort by number of previous resolved requests" %>
+        <%= f.check_box :sort_by %>
```

Comment:
Semantic can make a pretty checkbox with relatively little additional code, turning this:

<img src="https://user-images.githubusercontent.com/5882053/57055340-e2dc5580-6c68-11e9-9c84-7bd56bd31f02.png" height=50>

into this:

<img src="https://user-images.githubusercontent.com/5882053/57055314-b6283e00-6c68-11e9-8c5a-df35d2cd8934.png" height=50>

We do this here: https://github.com/mterwill/office-hours-help-queue/blob/fce98029f1a1576964d0c0df7ae0f128762a5dab/app/views/admin/course_queues/_form.html.erb#L29-L33

Reply:
Changed to drop down menu based on enum, see above comment and photo.